src/test/ui/async-await-drop-order-locals-are-hidden.rs

```diff
@@ -0,0 +1,32 @@
+// aux-build:arc_wake.rs
+// edition:2018
+
+#![allow(unused_variables)]
+#![feature(async_await, await_macro)]
+
+extern crate arc_wake;
+
+use arc_wake::ArcWake;
+use std::future::Future;
+use std::sync::Arc;
+use std::task::Context;
+
+struct EmptyWaker;
+
+impl ArcWake for EmptyWaker {
+    fn wake(self: Arc<Self>) {}
+}
+
+async fn foobar_async(x: u32, (a, _, _c): (u32, u32, u32), _: u32, _y: u32) {
+    assert_eq!(__arg1, (1, 2, 3)); //~ ERROR cannot find value `__arg1` in this scope [E0425]
+    assert_eq!(__arg2, 4); //~ ERROR cannot find value `__arg2` in this scope [E0425]
+}
+
+fn main() {
```

Comment:
I think this test could just have `fn main()` and thus we don't need the `arc_wake` dep etc

Reply:
Fixed.